src/status_im/contexts/wallet/create_account/new_keypair/backup_recovery_phrase/view.cljs

```diff
@@ -44,8 +45,14 @@
                                         :2 false
                                         :3 false})
         revealed?                     (reagent/atom false)
-        {:keys [customization-color]} (rf/sub [:profile/profile])]
+        {:keys [customization-color]} (rf/sub [:profile/profile])
```

Comment:
I think there's sub for just the color and it's better to pull in just the data needed? 
cc @clauxx

Reply:
Yep, otherwise we'll re-render every time there's changes in profile

Reply:
@clauxx do you know the sub I'm referring to? I can't remember the keyword for it

Reply:
must be this `:profile/customization-color`